UI/index.html

```diff
@@ -7,13 +7,20 @@
 		<script src="libs/jquery.tmpl.min.js" type="text/javascript"></script>
 		<script src="libs/jquery-ui.min.js" type="text/javascript"></script>
 		<link href="libs/jquery-ui.min.css" rel="stylesheet"></link>
+		<script src="libs/moment.js" type="text/javascript"></script>
 		<script src="../libs/ie11CustomProperties.js"></script>
 		<script src="../libs/lodash.min.js"></script>
 		<script src="../libs/d3.v4.min.js"></script>
 		<script src="../libs/KoreGraphAdapter.js" type="text/javascript"></script>
 		<script src='../libs/anonymousassertion.js'></script>
 		<script src="../kore-bot-sdk-client.js"></script>
 		<script src="chatWindow.js" type="text/javascript"></script>
+		<script src="libs/jquery.daterangepicker.js" type="text/javascript"></script>
```

Comment:
@VarshaPo-Kore  these changes need to go in index_widgets_chat.html(chat window with widgets) as well